doc/users/whats_new.rst

```diff
@@ -453,65 +479,141 @@ figure where the subplots will be created. If ``fig`` is ``None`` (default)
 then the method will use the current figure retrieved by
 :func:`~matplotlib.pyplot.gcf`.
 
-Example
-~~~~~~~
+
 ::
 
     subplot2grid(shape, loc, rowspan=1, colspan=1, fig=myfig)
 
 
-Interpolation in fill_betweenx
-------------------------------
+Interpolation in ``fill_betweenx``
+----------------------------------
 
 The ``interpolate`` parameter now exists for the method
-:func:`~matplotlib.axes.Axes.fill_betweenx`. This allows a user to interpolate
-the data and fill the areas in the crossover points, similarly to
-:func:`~matplotlib.axes.Axes.fill_between`.
+:func:`~matplotlib.axes.Axes.fill_betweenx`. This allows a user to
+interpolate the data and fill the areas in the crossover points,
+similarly to :func:`~matplotlib.axes.Axes.fill_between`.
 
 
-Validation of line style rcParams
----------------------------------
+New keyword argument ``sep`` for EngFormatter
+---------------------------------------------
+
+A new ``sep`` keyword argument has been added to
+:class:`~matplotlib.ticker.EngFormatter` and provides a means to
+define the string that will be used between the value and its
+unit. The default string is ``" "``, which preserves the former
+behavior. Additionally, the separator is now present between the value
+and its unit even in the absence of SI prefix.  There was formerly a
+bug that was causing strings like ``"3.14V"`` to be returned instead of
+the expected ``"3.14 V"`` (with the default behavior).
+
+Extend ``MATPLOTLIBRC`` behavior
+--------------------------------
+
+The environmental variable can now specify the full file path or path
```

Comment:
full file path or directory path? (the sentence is actually not 100% clear although it's not too hard to guess)